integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
That is a reasonable excuse to include a log message here, but it should still conform to `log.error(e, "message")`